addons/website/views/website_templates.xml

```diff
@@ -209,10 +209,30 @@
         </t>
     </xpath>
 
+    <!-- Breadcrumb -->
+    <xpath expr="//main/*[1]" position="before">
+        <t t-if="'has_parent_page' in main_object and main_object['has_parent_page']">
+            <div class="page_breadcrumb" data-name="Breadcrumb">
+                <t t-set="parent_page" t-value="main_object.parent_page_id"/>
+                <section>
+                    <div class="container o_not_editable py-2">
+                        <t t-if="parent_page.parent_page_id">
```

Comment:
Don't you think this condition should be above where you are checking ```has_parent_page``` or can you elaborate this whole UI changes like in which case we want to display ```parent_page.url``` etc.

Reply:
In t-set, I am storing the reference for the parent page for easy access. As for condition, to show up to 3 levels of parents in breadcrumb checking that the current pages' parent also has a parent page then only UI should change.